.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'daily'
+    commit-message:
+      prefix: '⬆️ deps:'
+      prefix-development: '⬆️ deps(dev):'
+    assignees:
+      - bonustrack
+      - samuveth
+    reviewers:
```

Comment:
Should we add @ChaituVR and @mktcode ?

Reply:
If ok I'll add 😄

Reply:
2 reviewer is enough, it should be Sam and Mkt